fabfile.py

```diff
@@ -384,41 +404,25 @@ def restart_nginx():
     local('sudo service nginx restart')
 
 
-def write_cron_backup(new_env):
-    """ Creates a cron job that copies a file to a remote server
-    """
-    print("Writing cron {}_backup".format(PROJECT_NAME))
-    template = jinja_env.get_template('etc/conf_templates/cron_backup.jinja2')
-    fabfile = os.path.abspath(__file__).rstrip("c")  # pycs won't cut it
-    output = template.render(
-        fabric_file=fabfile,
-        virtualenv=new_env.virtual_env_path,
-        branch=new_env.branch,
-        unix_user=UNIX_USER
-    )
-    cron_file = "/etc/cron.d/{0}_backup".format(PROJECT_NAME)
-    local("echo '{0}' | sudo tee {1}".format(
-        output, cron_file
-    ))
+def write_cron_jobs(new_env):
+    template_dir = "etc/cron_templates"
 
+    cron_jobs = os.listdir(template_dir)
```

Comment:
I would probably have done a glob match on e.g. template_dir + `*.jinja2` to avoid the case where for example a `.swp` file gets made a cron job by accident and we run everything twice at the same time.

Reply:
👍